### PR TITLE
Feature#6 add boostrap

### PR DESCRIPTION
**Pull request by LDVera**
Was added the bootstrap
boostrap was tested with the login page only to know that works the installation (this is not the final front-end)
image
****

### testing
follow the next steps and be sure to edit the `database.yml` file
you should have postgresql service active to test all

- rails db:create
- rails db:migrate
- rails s (to start the server)
- join to the `localhost:3000/sign_in`

### CODE REVIEW

File: app/views/tasks/_task.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1,2 @@
+json.extract! task, :id, :name, :description, :status, :due_to, :user_id, :created_at, :updated_at
+json.url task_url(task, format: :json)
```
Comment (MiguelOcegueraM) [inline app/views/tasks/_task.json.jbuilder:2]: ?

File: config/database.yml [hunk 2]
```diff
@@ -40,8 +32,9 @@ development:
   #username: Kata_todo_app_team_3
 
   # The password associated with the postgres role (username).
-
-  
+  database: ToDo_development
+  username: postgres 
+  password: password
```
Comment (MiguelOcegueraM) [inline config/database.yml:37]: remember the gitignore

File: db/migrate/20210505212905_create_tasks.rb [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
   def change
     create_table :tasks do |t|
       t.string :name
       t.string :description
       t.boolean :status
-      t.datetime :due
+      t.datetime :due_to
       t.references :user, null: false, foreign_key: true
 
       t.timestamps
```
Comment (MiguelOcegueraM) [inline db/migrate/20210505212905_create_tasks.rb:10]: nice

**Review by MiguelOcegueraM — DISMISSED** [review]
nice man!

File: app/views/main/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1> welcome <%= current_user.name%> </h1>
+<%= link_to "nueva tarea", new_task_path%>
```
Comment (OchoaHub) [inline app/views/main/index.html.erb:2]: In english please

**Review by OchoaHub — DISMISSED** [review]
I like you design ;)

**Review by jnegrete15 — APPROVED** [review]

**Review by LPMendoza — APPROVED** [review]

**Pull request merged**